tests/route/test_static_route.py

```diff
@@ -38,19 +42,40 @@ def is_dualtor(tbinfo):
 
 
 def add_ipaddr(ptfhost, nexthop_addrs, prefix_len, nexthop_devs, ipv6=False):
-    for idx in range(len(nexthop_addrs)):
-        if ipv6:
+    if ipv6:
+        for idx in range(len(nexthop_addrs)):
             ptfhost.shell("ip -6 addr add {}/{} dev eth{}".format(nexthop_addrs[idx], prefix_len, nexthop_devs[idx]), module_ignore_errors=True)
-        else:
-            ptfhost.shell("ip addr add {}/{} dev eth{}".format(nexthop_addrs[idx], prefix_len, nexthop_devs[idx]), module_ignore_errors=True)
+    else:
+        vlan_host_map = defaultdict(dict)
+        for idx in range(len(nexthop_addrs)):
+            mac = VLAN_BASE_MAC_PATTERN.format(idx)
```

Comment:
I think you could get the base mac from the ptf interface itself instead of predefined one. Can you please check?

Reply:
Thanks for this suggestion. Using the mac from the ptf interface instead.